src/Pages/Home.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../api";
+import ArticleCard from "../components/ArticleCard";
+import { Container, Row, Col } from "react-bootstrap";
+
+const Home = () => {
+  const [articles, setArticles] = useState(null);
```

Comment:
This initial setting should be of the same data type that you expect to set, therefore, if you're wanting to set this as an array, the initial value should be an empty array.